### ISSUE TITLE
Shouldn't the block be copied?

### ISSUE THREAD

**Issue by jwwalker** [jakob/NSAlertBlockMethods#3]
The block documentation says "You only need to make a copy when you expect the block to be used after destruction of the scope within which it was declared."  That seems to be the case here.  `beginSheetModalForWindow:modalDelegate:didEndSelector:contextInfo:` exits, allowing the block to go out of scope, before the completion block runs.

**Comment by jakob** [comment]
Good catch, I'm pretty sure you are right. That might cause some hard to fix bugs.
